src/styles/content.scss

```diff
@@ -335,4 +335,70 @@ table.os-raise-horizontaltable {
 .os-raise-extrasupport a {
   color: #0A3E5B;
   text-decoration: underline;
-}
\ No newline at end of file
+}
+
+.os-raise-accordion {
+  width: 100%;
+}
+
+.os-raise-accordion-item {
+  border-bottom: 1px solid #D9D9D9;
+}
+
+.os-raise-accordion-header {
+  display: flex;
+  align-items: center;
+  background-color: white;
+  padding: 0.625rem;
+  border-bottom: 1px solid #D9D9D9;
+  color: $os-dark-green;
+  font-size: 1rem;
+  font-weight: bold;
+  border: none;
+  width: 100%;
+  cursor: pointer;
```

Comment:
Randomly noticed this -- is it still necessary if we're using `<button>`?

Reply:
Ah -- I just realized that we do actually need it. Things will work in Moodle otherwise because of [this](https://github.com/moodle/moodle/blob/513f3b02c76e0321b3c57ca8014d3100b4da8d94/theme/boost/style/moodle.css#L13466), but we need to specify it here so it works on the OER site.